lib/project_types/script/messages/messages.rb

```diff
@@ -150,6 +150,11 @@ module Messages
           create: {
             select_extension_point: "Which extension point do you want to use?",
             script_name: "Script Name",
+
+            error: {
+              invalid_name: "You have used an invalid character in your script name. This includes "\
+                          "characters like \"<\", \">\", or \"/\". Choose a new name.",
```

Comment:
Sorry missed something - if we're using an 'allowed' set rather than not allowed, should this message be different? Like `You have used an invalid character in your script name. Valid characters include ...`. 

The user could type something outside of the allowed list that this message doesnt describe

Reply:
I like this idea, let me check in to see if Annette has any thoughts!

Reply:
I messaged Annette and she raised the concern that it's a little more ambiguous because developers have to take the ok character list and try to figure out what the invalid ones are from it.  So I think it might be reasonable to keep the message as it is.

Reply:
The allowed list is pretty straightforward, isn't it? Alphanumeric characters, space, underscore and hyphen? I agree with Jacob here. I'd rather not need to guess what is valid — just tell me what is ok.

Reply:
I've asked @abater1970 for her thoughts, and I believe she has checked in with the shopify content group channel to see if there is a consensus. I will update this when more information is available!

Reply:
Okay more information is available, it looks like it's best to orient around what we can do, so I will update the message to ```Invalid script name: replace or remove unsupported characters. Valid characters are numbers, letters, spaces, hyphens, or dashes.```

Reply:
Tiny point: A hyphen (‐) is a different character from a dash (– or —). The regexp only allow hyphens.